### ISSUE TITLE
[bug] malformed Python docstrings

### ISSUE THREAD

**Issue by alexcjohnson** [snehilvj/dash-mantine-components#421]
The DMC components have top-level descriptions that are just their names, for example:

https://github.com/snehilvj/dash-mantine-components/blob/6dcf035b31cb52e7ebdf30d43dae2942a66f51a5/src/ts/components/core/accordion/Accordion.tsx#L44-L45

The result in codegen is broken :

```py
class Accordion(Component):
    """An Accordion component.
ordion

Keyword arguments:
...
```
I suspect because we're expecting a multiline description here, and in a multiline docstring the first three characters are ` * `, eg dcc.CheckList:

```js
/**
 * Checklist is a component that encapsulates several checkboxes.
 * The values and labels of the checklist are specified in the `options`
 * property and the checked items are specified with the `value` property.
 * Each checkbox is rendered as an input with a surrounding label.
 */
export default class Checklist extends Component {

```

We should fix this on the component generation side (I'll make an issue, or maybe just a PR if it's simple), but wanted to flag it here because (a) this is where I saw it, and (b) perhaps it would be good to improve the DMC docstrings, giving each component more than just its name as a top-level description?

**Comment by AnnMarieW** [comment]
Yes, I noticed that too, but we don't use the description in the documentation so I didn't make it a priority.  It would be pretty tedious to fix all the components here.  I think I found where it was clipping the descriptions on the dash side and it shouldn't be too hard to fix there.

**Comment by AnnMarieW** [comment]
Nice update!  This will be helpful for everyone making custom Dash components :slightly_smiling_face:

**Comment by alexcjohnson** [comment]
Ha, didn't realize you could have a PR in a whole different org close an issue... anyway I guess it makes sense to close with my PR since it'll no longer be malformed. I do think having a one-sentence description on each component would be nice, so that in an IDE someone reading the code gets a little more context on what a given component is for. Plus as is it'll be redundant because Dash starts the docstring with "A(n) {component_name} component", then DMC components have the name again on the next line.

**Comment by AnnMarieW** [comment]
>  I do think having a one-sentence description on each component would be nice, so that in an IDE someone reading the code gets a little more context on what a given component is for. Plus as is it'll be redundant because Dash starts the docstring with "A(n) {component_name} component", then DMC components have the name again on the next line.

I agree!  I just reopened this issue :-) 

For anyone reading this thread --  this is a great first issue to work on, especially if you’re new to contributing to open source. You don’t need any React knowledge, so it’s an easy way to get familiar with the structure of this library and see how the development process works. I’m happy to help mentor anyone interested in making a PR!
